### DISCUSSION TITLE
Using (custom) editor blocks within custom editor blocks

### DISCUSSION THREAD

**Discussion by cyanonoob** [keystonejs/keystone#7509] [Questions] [answered]
Hi all, I posted this question in the slack a couple days ago but just noticed that's no longer the way, so reposting here. I'm hoping that's ok.

The actual question: is it also possible to use custom blocks within custom blocks in a document field? Or is recursion evil there?

**Comment by raveling** [comment, marked as answer]
@cyanonoob  it’s not currently possible, but we’re thinking about enabling it in the future. Hope this helps!
